crates/holochain/src/sweettest/sweet_zome.rs

```diff
@@ -19,3 +21,52 @@ impl SweetZome {
         &self.name
     }
 }
+
+#[derive(Default)]
+/// A helper for creating [`InlineZomeSet`]
+pub struct SweetEasyInline(pub InlineZomeSet);
```

Comment:
Haha you skipped right over Sweet* and went straight to SweetEasy*! It must be really good.

Reply:
Haha yeh it should probably be SweetInlineZome but I wanted to show that it is simplified by calling it easy.

Reply:
That's already what the Sweet things are all about. But it's fine.